static/app/views/performance/landing/widgets/widgets/vitalWidgetMetrics.tsx

```diff
@@ -104,19 +104,20 @@ export function VitalWidgetMetrics(props: PerformanceWidgetProps) {
             project={project}
             environment={environment}
             query={new MutableSearch(eventView.query)
-              .addFilterValues('transaction', [
-                widgetData.list.data[selectedListIndex].transaction,
-              ])
+              .addFilterValues(
+                'transaction',
+                widgetData.list.data.map(listItem => listItem.transaction) as string[]
```

Comment:
I tried this with `?datasource=snuba` and it looks like this translates to the query
```
transaction:/api/{:orgId}/zap transaction:/api/{:orgId}/foo transaction:/api/{:orgId}/bar
```
which is equivalent to 
```
transaction:/api/{:orgId}/zap AND transaction:/api/{:orgId}/foo AND transaction:/api/{:orgId}/bar
```

Can we change this to `transaction IN [...]`?

Reply:
`?datasource=mock` basically ignores the query, which is why it did not catch this.